app/views/components/docs/error-summary.yml

```diff
@@ -0,0 +1,26 @@
+name: Form error summary
+description: Used at the top of the page, to summarise validation errors.
+accessibility_criteria: |
+  - should be focused on page load
+  - links that reference errors should focus what they link to
```

Comment:
Sorry for being dense, but what does this mean?

Reply:
Not being dense, it's badly worded, I'll try and improve this.